tests/apps/test_embed.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Unit tests for strawberryfields.apps.train.embed
+"""
+# pylint: disable=no-self-use,unused-argument
+import numpy as np
+import pytest
+from strawberryfields.apps.train import embed
+
+feats = [
+    [[0.1, 0.2], [0.2, 0.1]],
+    [[0.1, 0.2, 0.3], [0.3, 0.1, 0.2], [0.2, 0.3, 0.1]],
+    [[0.1, 0.2, 0.3, 0.4], [0.4, 0.1, 0.2, 0.3], [0.3, 0.4, 0.1, 0.2], [0.2, 0.3, 0.4, 0.1]],
+]
```

Comment:
Good idea! Sure, I can do that.

Reply:
Just in case it gets missed (sometimes I miss the main comment):
Here's an example test that might work
```python
@pytest.mark.parametrize("m", [3, 4, 5])
@pytest.mark.parametrize("d", [3, 4, 5])
def test_exp_features_integration(m, d):
    """Test that ``strawberryfields.apps.train.embed.ExpFeatures`` outputs weights and jacobian 
    of the correct shape"""
    features = np.ones((m, d))
    expf = embed.ExpFeatures(features)
    features = np.ones(d)

    assert expf(features).shape == (m,)
    assert expf.jacobian(features).shape == (m, d)
```